packages/create-instance/create-slot-vnodes.js

```diff
@@ -18,22 +20,41 @@ function createVNodesForSlot (
   const el =
     typeof slotValue === 'string' ? compileToFunctions(slotValue) : slotValue
 
-  const vnode = h(el)
+  let vnode = h(el)
+  if (typeof slotValue === 'string') {
+    const vue = new Vue()
+    const _vue = new _Vue()
+    for (const key in _vue._renderProxy) {
+      if (!(vue._renderProxy[key])) {
+        vue._renderProxy[key] = _vue._renderProxy[key]
```

Comment:
Why not always use _vue.renderProxy?

Reply:
Do you mean when `_vue.renderProxy` is used?

Reply:
It does not raise the error.

Reply:
I am sorry.
Yes, I do.